app/src/main/res/layout/layout_switch_drive.xml

```diff
@@ -15,12 +15,13 @@
   ~ You should have received a copy of the GNU General Public License
   ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   -->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="120.8dp"
-    android:gravity="bottom"
+    android:paddingStart="16dp"
```

Comment:
You can use `marginStandardMedium` for that no?

Reply:
This value here should truly never change no matter what our `marginStandardMedium` actual value becomes, so it's hard to say